### DISCUSSION TITLE
Is it worth starting a project with yowsup in 2020?

### DISCUSSION THREAD

**Discussion by KaratekHD** [tgalal/yowsup#3029] [Q&A]
Does yowsup currently work? How about this banning issue? Will my number get banned when trying to sign up?

**Comment by endroemily** [comment]
> Does yowsup currently work? How about this banning issue? Will my number get banned when trying to sign up?

Same issue, did you find any solution?

**Reply by KaratekHD** [reply]
Nope, sadly didn't. All I found is #2829

**Reply by endroemily** [reply]
Somebody can register without any problem, but I don't know how they did it

**Reply by KaratekHD** [reply]
Do you know the name of them?

**Reply by ben221199** [reply]
I did, but I used a number that someone gave to me. I registered, he sent me the SMS code and I verified my registration. At the moment, my yowsup client is still not banned.

**Reply by juangarlop** [reply]
> Somebody can register without any problem, but I don't know how they did it

Yes, can i help you? i stay in next step but i cannot get connected...

**Comment by jess242015** [comment]
I want to grab a What‘sapp to read me. Is that possible? 

Von meinem iPhone gesendet

**Reply by ben221199** [reply]
Uhm...

**Comment by jess242015** [comment]
Okay, I thought it wouldn't be that easy! And not quite allowed! 

Von meinem iPhone gesendet

**Reply by ben221199** [reply]
What do you mean?

**Comment by jess242015** [comment]
I want read what’s app to my Boyfriend! 

Von meinem iPhone gesendet

**Reply by ben221199** [reply]
I think you are not serious, tho. And by the way, to keep it organized here, it would be nice if you reply in the thread.

**Comment by baidwan007** [comment]
In the cli section i saw that we could use [/presence subscribe <contact>] to subscribe to presence of a contact. Does this mean that we ll be updated when the user comes online and when he goes offline?

**Reply by baidwan007** [reply]
Infact, my usecase revolves around knowing the online status of contacts. Is there a way i can use yousup to achieve this?

**Reply by dshetyo** [reply]
@baidwan007 Did u find a way to fetch online status of contact?
